Projects/My_website/myWebsite/blog/views.py

```diff
@@ -26,13 +32,31 @@ def get_queryset(self):
         return data
 
 
-class blogComments(APIView):
-    def get(self, request, id):
-        data = Comments.objects.filter(blog=id)
-        serializer = CommentSerializer(data, many=True)
-        return Response(serializer.data)
+# class to get comments of a specific blog
+class blogComments(viewsets.ModelViewSet):
```

Comment:
@codal-uchoudhary Class name should always start with caps. this should be `BlogComments` . Please update at all places